src/common/uis.ts

```diff
@@ -33,7 +33,7 @@ export async function fetchUIs(imageVersion: string): Promise<void> {
     return;
   }
 
-  const sourcePath = `${uis.remoteAssets}${version}.tgz`;
+  const sourcePath = `${uis.remoteAssets}${version}-uis.tgz`;
```

Comment:
Since I uploaded the snapshots here I added the old file with the new name.

Its was a bit confusing to know what is what without this, and there is only 1 file needing the duplicate name now.